### PR TITLE
Fix css & js dependencies (vue) in package.json

### PR DESCRIPTION
**Pull request by mjauvin**
- removed `max-w-6` class in `_menu-side.php` partial which broke menu width.
- set bgcolor for clear-input-text in light mode to hide the search icon behind it.
- update vue dependency in package.json to match what is used in core

### CODE REVIEW

**Comment by mjauvin** [comment]
Many thanks to @jaxwilko to help setup docker for testing and resolving the CSS issue!

**Comment by LukeTowers** [comment]
Do you have screenshots?

**Comment by mjauvin** [comment]
> Do you have screenshots?

Before:
image
After:
image

File: assets/css/src/custom.css [hunk 1]
```diff
@@ -124,6 +124,12 @@ div[data-control='media-manager-crop-tool'] {
     }
 }
 
+.toolbar-widget {
+  .clear-input-text {
+    background-color: white;
+  }
```
Comment (LukeTowers) [inline assets/css/src/custom.css:130]:
```suggestion
    .clear-input-text {
      background-color: white;
    }
```

**Review by LukeTowers — APPROVED** [review]

**Pull request merged**